.github/contributing.md

```diff
@@ -0,0 +1,38 @@
+# Contributing
+
+## Security
+
+If you think you've found a **security issue**, please do not mention it in this repository.
+Instead, email renovate-disclosure@whitesourcesoftware.com with as much details as possible so that it can be handled confidentially.
```

Comment:
```suggestion
```

Question: do we want to redirect security bugs in the documentation website to email as well?

Reply:
Yes, just in case